src/python/pants/engine/subsystem/native_engine_version

```diff
@@ -1 +1 @@
-06e4c8ac8a3cf4f20cb74929208f35ab4fe75d0b
```

Comment:
This change is causing 3 CI shards to fail. You might revert the change to this file and/or rebase against master, then confirm by running pants, something like:
```
rm -rf ~/.cache/pants/bin/native-engine/
./pants
```

That should re-build the native engine locally and it should cause no change to the contents of `src/python/pants/engine/subsystem/native_engine_version`.